models/genres.js

```diff
@@ -0,0 +1,9 @@
+const genres = (connection, sequelize) => {
+  return connection.define('genres', {
+    id: { type: sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+    genre: { type: sequelize.STRING, allowNull: false },
```

Comment:
I'd have called this `name`. A `findOne` from this novel will return a `genre` and `genre.genre` is weird.